lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -409,7 +409,7 @@ public function getMaxResults()
      * 'groupBy', 'having' and 'orderBy'.
      *
      * @param string  $sqlPartName
-     * @param string  $sqlPart
+     * @param mixed  $sqlPart
```

Comment:
Yes I have seen but I thought that you can call this function with a string or with an array so I thought that mixed type is better than string inside documentation.
If It's better `string` I understand but there are two ways for me to call this function for that parameter